documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -6,15 +6,17 @@ Cutting data of interest from SQW files and objects
 
 .. |Q| replace:: :math:`|\textbf{Q}|`
 
-Horace stores the data produced in a neutron experiment in a 4-D dataset
-which are called the
-"pixels". These represent the neutron events measured during an
```

Comment:
4-D dataset -- its incorrect -- events are provided in 1D dataset. More detailed description of the structure of sqw object is added